portal/src/OnboardingSurveyLayout.tsx

```diff
@@ -61,13 +93,20 @@ export default function SurveyLayout(
     secondaryButton,
     children,
   } = props;
+  const theme = useTheme();
+  const bodyStyles = useMemo(() => {
+    return {
+      backgroundColor: theme.semanticColors.bodyStandoutBackground,
+      height: "100vh",
+    };
+  }, [theme]);
   return (
-    <div className={styles.root}>
+    <div style={bodyStyles}>
```

Comment:
We use className to customize CSS for non-fluent UI element. Move height to the CSS file.